docs/explanations/how_to_interpret_unattended_upgrades_is_running.md

```diff
@@ -0,0 +1,81 @@
+unattended-upgrades on the machine. The expected output follows this json example:
+
+```json
+{
+  "_schema_version": "v1",
+  "data": {
+    "attributes": {
+      "apt_periodic_job_enabled": true,
+      "package_lists_refresh_frequency_days": 1,
+      "systemd_apt_timer_enabled": true,
+      "unattended_upgrades_allowed_origins": [
+        "${distro_id}:${distro_codename}",
+        "${distro_id}:${distro_codename}-security",
+        "${distro_id}ESMApps:${distro_codename}-apps-security",
+        "${distro_id}ESM:${distro_codename}-infra-security"
+      ],
+      "unattended_upgrades_disabled_reason": null,
+      "unattended_upgrades_frequency_days": 1,
+      "unattended_upgrades_last_run": null,
+      "unattended_upgrades_running": true
+    },
+    "meta": {
+      "environment_vars": [],
+      "raw_config": {
+        "APT::Periodic::Enable": "1",
+        "APT::Periodic::Unattended-Upgrade": "1",
+        "APT::Periodic::Update-Package-Lists": "1",
+        "Unattended-Upgrade::Allowed-Origins": [
+          "${distro_id}:${distro_codename}",
+          "${distro_id}:${distro_codename}-security",
+          "${distro_id}ESMApps:${distro_codename}-apps-security",
+          "${distro_id}ESM:${distro_codename}-infra-security"
+        ]
+      }
+    },
+    "type": "UnattendedUpgradesStatus"
+  },
+  "errors": [],
+  "result": "success",
+  "version": "27.14~16.04.1",
+  "warnings": []
+}
+```
+
+As we can see from this output, we have a variable named `unattended_upgrades_running`. That variable
+indicates if unattended-upgrades is properly configured and running on the machine.
+That value of this field will only be `true` if *ALL*  of the following prerequisites are also true:
+
+* *`apt_periodic_job_enable` is true*: That variable indicates if the APT::Periodic::Enable configuration variable
+  is turned on. If it is turned off, unattended-upgrades will not automatically run on the machine.
+* *`package_lists_refresh_frequency_days` is non-zero*:  That variable shows the value of APT::Periodic::Package-List-Frequency.
+  This configuration defines the daily frequency for updating package sources in the background. If it has a zero value, this step will never
+  happen and unattended-upgrades might not be able to install new version of the packages.
+* *`systemd_apt_timer_enabled` is true*: That variable is true if both `apt-daily.timer` and `apt-daily-upgrade.timer` are running
+  in the machine. These timers are the ones that control when unattended-upgrades run. The first job, `apt-daily.timer` is responsible
+  for triggering the package source updates while `apt-daily-upgrade.timer` is responsible for running unattended-upgrades to download
+  the packages. If one of these jobs is disabled, unattended-upgrades might not work as expected.
+* *`unattended_upgrades_allowed_origins` is not empty*: That variable define the origins that
```

Comment:
```suggestion
* *`unattended_upgrades_allowed_origins` is not empty*: This variable defines the origins that
```

Reply:
Also, I'm not sure what origins means in this context. E.g. what's the expected/allowed format?

Reply:
We are displaying the format of those origins on the example we are providing. But let me know if I should further explain that

Reply:
I think it should be fine as is, I just wanted to clarify